### ISSUE TITLE
Extension deletes first 1000 comments sorted by `New`, leaving comments under `Hot`, `Top`, and `Controversial` sortings

### ISSUE THREAD

**Issue by travisreed-dfw** [sr33/ares#3]
The extension will delete the first 1000 comments when sorted by `New`    
This appears to be everything, but if you sort again by `Hot`, `Top` or `Controversial` you will see much older comments are still there. 

Sorting by `New` does not reload these comments. 

```
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
?efe2d409a42=f9ce4f81e6326:1 A cookie associated with a resource at http://google.com/ was set with `SameSite=None` but without `Secure`. A future release of Chrome will only deliver cookies marked `SameSite=None` if they are also marked `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5633521622188032.
ErrorHandling.js:44 Error: No slots provided to apstag.fetchBids
    at xe (apstag.js:3134)
    at apstag.js:3029
    at Array.map (<anonymous>)
    at Be (apstag.js:3027)
    at ErrorHandling.js:116
    at b (GDPR.js:421)
    at GDPR.js:347
    at e (GDPR.js:283)
    at t.GDPR (GDPR.js:276)
    at Le (apstag.js:3006)
m @ ErrorHandling.js:44
UI.ts:34 --: Nuke Reddit History initiated :--

```

Google Chrome is up to date
Version 78.0.3904.108 (Official Build) (64-bit)

**Comment by travisreed-dfw** [comment]
Same results in Firefox 70.0.1 (64-bit). 



```
Error: "No slots provided to apstag.fetchBids"
    xe apstag.js:3134
    Be apstag.js:3029
    Be apstag.js:3027
    wrapFunction ErrorHandling.js:116
    b GDPR.js:421
    GDPR GDPR.js:347
    e GDPR.js:283
    GDPR GDPR.js:509
    Le apstag.js:3006
    Ue apstag.js:3307
    qe apstag.js:3316
    <anonymous> (index):9
    push pubads_impl_2019111401.js:1
    nl pubads_impl_2019111401.js:1
    R pubads_impl_2019111401.js:1
    Oq pubads_impl_2019111401.js:1
    <anonymous> pubads_impl_2019111401.js:1
    <anonymous> pubads_impl_2019111401.js:1
ErrorHandling.js:44:43
Request to access cookie or storage on “<URL>” was blocked because it came from a tracker and content blocking is enabled. 4
--: Nuke Reddit History initiated :-- UI.ts:34:24

```

**Comment by travisreed-dfw** [comment]
Issue persists with version 2.1 of the extension found [here](https://old.reddit.com/r/NukeRedditHistory/comments/cp5b0k/nuke_reddit_history_210_%CE%B2_with_posts_deletion_is/)

**Comment by sr33** [comment]
Thanks for reporting this. It's a known issue and I'm working to fix it.

**Comment by sr33** [comment]
Fixed this in `version 2.2.0`

Updates available on the Chrome Extension Store & Firefox Add-On Store

**Comment by SylveonBottle** [comment]
Having the same issue, `2.3.1`.
Apparently the extension searches for:
```
sort: new
sort: hot
sort: controversial
sort: controversial t: hour
sort: controversial t: day
sort: controversial t: week
sort: controversial t: month
sort: controversial t: year
sort: controversial t: all
sort: top
sort: top t: hour
sort: top t: day
sort: top t: week
sort: top t: month
sort: top t: year
sort: top t: all
```

However, reddit does not show you the full comments, so ares does not delete all comments and requires the page to be refreshed every 3 minutes to continue deleting comments.

Maybe add an auto-refresh page to verify that the comments have been deleted? Or was this extension made with the assumption that the user has RES and it's infinite scrolling feature..?
